### PR TITLE
new rule: Certificate Exported

### PR DESCRIPTION
**Pull request by YamatoSecurity**
### Summary of the Pull Request

This PR adds a rule to detect when an attacker uses a tool like mimikatz or powershell to export certificates and/or their private keys.

### Detailed Description of the Pull Request / Additional Comments

No extra info.

### Example Log Event

```
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
<System>
  <Provider Name="Microsoft-Windows-CAPI2" Guid="{5bbca4a8-b209-48dc-a8c7-b23d3e5216fb}" /> 
  <EventID>70</EventID> 
  <Version>0</Version> 
  <Level>4</Level> 
  <Task>70</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x4000000000000080</Keywords> 
  <TimeCreated SystemTime="2020-07-11T13:21:17.5140762Z" /> 
  <EventRecordID>13969094</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="5708" ThreadID="5712" /> 
  <Channel>Microsoft-Windows-CAPI2/Operational</Channel> 
  <Computer>wec02</Computer> 
  <Security UserID="S-1-5-21-1153173314-1076311963-3278442693-500" /> 
 </System>
 <UserData>
  <CryptAcquireCertificatePrivateKey>
  <Certificate fileRef="3CD6B0EFAF68549EFE9ED2316426FCD7FF81A6A8.cer" subjectName="wec02.offsec.lan" /> 
  <Flags value="10000" CRYPT_ACQUIRE_ALLOW_NCRYPT_KEY_FLAG="true" /> 
  <EventAuxInfo ProcessName="mimikatz.exe" /> 
  <CorrelationAuxInfo TaskId="{4254EC74-97ED-4EF5-A12B-8CB3F120BDD1}" SeqNumber="2" /> 
  <Result value="0" /> 
  </CryptAcquireCertificatePrivateKey>
 </UserData>
</Event>
```

### Fixed Issues

No fixes to anything existing.

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules-placeholder/windows/win_certificate_exported.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    product: windows
+    definition: 'The CAPI2/Operational log needs to be enabled as it is disabled by default. Powershell ScriptBlock logging should also be enabled.'
+detection:
+    select_CAPI_log:
+        Channel: Microsoft-Windows-CAPI2/Operational
+        EventID: 70 # "Acquire Certificate Private Key"
+    select_CS_client_log:
+        Channel: Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational
+        EventID: 1007 # "A certificate has been exported"
+    select_PwSh_log:
```
Comment (frack113) [inline rules-placeholder/windows/win_certificate_exported.yml:22]: Can be an update to the rule [aa7a3fce-bef5-4311-9cc1-5f04bb8c308c](https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml)
category: ps_script cover the channel and EventID4104
detection is allready for Export-PfxCertificate

Comment (YamatoSecurity) [reply rules-placeholder/windows/win_certificate_exported.yml]: Adding to the [aa7a3fce-bef5-4311-9cc1-5f04bb8c308c](https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml) rule for powershell export-certificate sounds good to me. I removed that part of the rule.

**Comment by YamatoSecurity** [comment]
If https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml is a level `high`, then should I mark this as level `high` as well?

**Comment by nasbench** [comment]
> If https://github.com/SigmaHQ/sigma/blob/master/rules/windows/powershell/powershell_script/posh_ps_susp_export_pfxcertificate.yml is a level `high`, then should I mark this as level `high` as well?

The PowerShell one shouldn't be high in the first place. I reduced it to medium.

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]

**Review by phantinuss — APPROVED** [review]

**Pull request merged**